hanko-js/src/lib/HankoClient.ts

```diff
@@ -131,6 +179,11 @@ class HttpClient {
     });
   }
 
+  // _post2 part of a workaround for a Safari bug. See _fetch2().
```

Comment:
Doc comment indicates the name of the function is `_post2` but it is `post2`.

```suggestion
  // post2 part of a workaround for a Safari bug. See _fetch2().
```

or rename the function to `_post2`.